src/highdicom/sr/content.py

```diff
@@ -938,15 +989,15 @@ def __init__(
     @classmethod
     def from_seg_dataset(
```

Comment:
Given that we call the related methods `from_source_image()` and `from_source_value_map()`, I wonder whether we should call this method `from_segmentation()` for consistency. What do you think?

Reply:
I asked you the same question above! I'll make the change

Reply:
Changed as suggested in 9aca15a @hackermd

Reply:
> I asked you the same question above! I'll make the change

I just realized that, too. We are on the same page 👍